### PR TITLE
Category carousel

### PR DESCRIPTION
**Pull request by ileenshaar**
I have created the carousel component with an array of fake data for testing

### CODE REVIEW

**Review by mahmoud-alragabi — CHANGES_REQUESTED** [review]
Please resolve the conflicts first, so i can do a proper review

File: src/App.jsx [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+function App() {
+  return (
+    <Box
+      sx={{
+        width: "100%",
+        height: "100%",
+        position: "absolute",
+      }}
+    >
+      {/* Header goes here */}
```
Comment (mahmoud-alragabi) [inline src/App.jsx]: Remove this comment

File: src/components/Carousel/Carousel.jsx [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+      }}
+    >
+      <AutoPlaySwipeableViews
+        index={activeStep}
+        onChangeIndex={handleStepChange}
+        enableMouseEvents
+      >
+        {images.map((step, index) => (
+          <div key={step.label}>
+            {Math.abs(activeStep - index) <= 2 ? (
```
Comment (mahmoud-alragabi) [inline src/components/Carousel/Carousel.jsx]: Should be abstracted to a variable

File: src/components/Carousel/Carousel.jsx [hunk 3]
```diff
@@ -0,0 +1,139 @@
+import * as React from "react";
```
Comment (mahmoud-alragabi) [inline src/components/Carousel/Carousel.jsx:1]: Needs more abstraction, you need to break it down to smaller components

**Review by mahmoud-alragabi — CHANGES_REQUESTED** [review]

File: src/components/Carousel/Carousel.jsx (hunk 2 above)
Comment (ileenshaar) [reply src/components/Carousel/Carousel.jsx]: I have abstracted the styles into a variable. Is that what you meant?

Comment (mahmoud-alragabi) [reply src/components/Carousel/Carousel.jsx]: No the equation that starts with math abs what does it mean ? Define it by a variable, someone reading your code doesn't have to analyse what that means they can just look at the var name and know

**Pull request closed without merging**